nucleus/common/internal-api/src/main/java/org/glassfish/internal/data/ApplicationInfo.java

```diff
@@ -372,6 +370,24 @@ public void start(
         }
     }
 
+    public void reload(
+            ExtendedDeploymentContext context,
+            ProgressTracker tracker) throws Exception {
+
+        StructuredDeploymentTracing tracing = StructuredDeploymentTracing.load(context);
+        DeploymentSpan span = tracing.startSpan(DeploymentTracing.AppStage.RELOAD);
+
+        super.reload(context, tracker);
+        // registers all deployed items.
+        for (ModuleInfo module : getModuleInfos()) {
+            DeploymentSpan innerSpan = tracing.startSpan(TraceContext.Level.MODULE, module.getName(), DeploymentTracing.AppStage.RELOAD);
```

Comment:
Should this innerspan not nested within Reload span (from line 378) so that we have proper reporting?